From b23c1f807cb20a9a4c42b63c3d80f63b045a43ef Mon Sep 17 00:00:00 2001
From: Xingyu Wu <xingyu.wu@starfivetech.com>
Date: Thu, 27 Apr 2023 15:44:00 +0800
Subject: watchdog: starfive: Fix the warning of starfive_wdt_match

Drop the function of of_match_ptr() to fix the warning of unused variable
'starfive_wdt_match'.

Fixes: db728ea9c7be ("drivers: watchdog: Add StarFive Watchdog driver")
Signed-off-by: Xingyu Wu <xingyu.wu@starfivetech.com>
Reviewed-by: Guenter Roeck <linux@roeck-us.net>
Link: https://lore.kernel.org/r/20230427074400.55380-1-xingyu.wu@starfivetech.com
Signed-off-by: Guenter Roeck <linux@roeck-us.net>
Signed-off-by: Wim Van Sebroeck <wim@linux-watchdog.org>
---
 drivers/watchdog/starfive-wdt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/watchdog/starfive-wdt.c b/drivers/watchdog/starfive-wdt.c
index 8f378900624de..8058fca4d05d3 100644
--- a/drivers/watchdog/starfive-wdt.c
+++ b/drivers/watchdog/starfive-wdt.c
@@ -595,7 +595,7 @@ static struct platform_driver starfive_wdt_driver = {
 	.driver = {
 		.name = "starfive-wdt",
 		.pm = &starfive_wdt_pm_ops,
-		.of_match_table = of_match_ptr(starfive_wdt_match),
+		.of_match_table = starfive_wdt_match,
 	},
 };
 module_platform_driver(starfive_wdt_driver);
-- 
cgit 

